### ISSUE TITLE
MySQL error on achievement 1489

### ISSUE THREAD

**Issue by BarbzYHOOL** [Sarjuuk/aowow#151]
Found this error on two sites using aowow (maybe outdated), here is the error:

```
Array
(
    [code] => 1064
    [message] => You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1
    [query] => SELECT criteria_id AS ARRAY_KEY, type AS ARRAY_KEY2, value1, value2, ScriptName FROM achievement_criteria_data WHERE criteria_id IN ()
    [context] => /membri/wowgaming/aowow/pages/achievement.php line 264
)
```

**To reproduce, go to:**
?achievement=1489

**example:**
https://wotlk.evowow.com/?achievement=1489

**Comment by Sarjuuk** [comment]
The fuck...? 
Why is this site running in debug mode.
Why the fuck is it placing ads.

**Comment by BarbzYHOOL** [comment]
Lol this site (evowow) is the most used site when people asks for a wotlk db, just saying :P

The other is the one for azerothcore project (but still using TC structure for now, so unmodified) https://wowgaming.altervista.org/aowow/?achievement=1489  but I'm certain it's outdated hence why I didn't link it

**Comment by Kittnz** [comment]
My 1 year old version is showing the achievement. So must be some change recently.

**Comment by Sarjuuk** [comment]
Nahhh..

You probably just disabled debug mode on your live site.
If the achievement doesn't have any criteria, you too have this error.

**Comment by Sarjuuk** [comment]
fixed in 470498f63c53366342b1e718511f4e80f8a109fd
